apps/chameleon/tailwind.config.js

```diff
@@ -21,6 +21,8 @@ module.exports = {
     },
     extend: {
       colors: {
+        'dim-black': '#191919',
```

Comment:
Let's remove this value in favor of a tailwind default black; find one on the colors page that is close, and remove this value from the code base. https://tailwindcss.com/docs/customizing-colors